src/css/import-resource-tree.css

```diff
@@ -42,6 +42,17 @@
     padding-top: 8px;
 }
 
+.import-resource-table .even-row-start .cell,
+.import-resource-table .odd-row-start .cell {
+    padding-top: 12px;
+}
+
+.import-resource-table .even-row-end .cell,
+.import-resource-table .odd-row-end .cell {
+    padding-top: 4px;
+    padding-bottom: 12px;
```

Comment:
Why these additional paddings were needed? Is by request by someone?

Reply:
Yes. After conversation with Donka.